src/content/getting-started/rbac-and-psp/index.md

```diff
@@ -484,6 +501,7 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ```
 
+
 ### Running applications that need privileged access
```

Comment:
This section is about binding a PSP to an SA to allow a workload to use it -- I'd suggest removing it entirely if that's cool with the reviewers

Reply:
I've removed this section. Holler at me if someone wants it to stay